.docker/build/config.json

```diff
@@ -8,14 +8,16 @@
     "GithubCommitUrl" : "https://github.com/FStarLang/FStar/commit",
     "OnDemandBuildDefinition" : "FStar\\FStar-{agentOS}",
 
-    "DockerFile": ".docker/build/{agentOS}/Dockerfile",
+    "DockerStyle": "run",
+    "DockerFile": ".docker/build/{agentOS}/Dockerfile.build",
     "DependencyFiles" : [
         ".docker/build/build_helper.sh",
         ".docker/build/build.sh"
     ],
+    "DockerCommand": "sudo docker",
 
     "DockerCacheEnabled" : true,
-    "CIBuildTarget" : "hacl-ci",
+    "CIBuildTarget" : "hacl-ci-build",
```

Comment:
The benchmarks segfault on Linux for me. As far as I can see they require specific CPU instructions that are missing or not properly compiled in.
As before, I'm fine with keeping `hacl-ci` the default here. This is what I'd run locally and what we'd run in Mozilla CI.

Reply:
That's definitely something that we need to look at, because EverCrypt is supposed to avoid this by detecting CPU features.

One of the concerns I have about building in a local checkout, is that the artifacts (especially binaries) that are built in a Docker container might not be compatible with the local system. This could be an instance of this.

Reply:
The issue is that the C performance tests do not go through the EverCrypt API and as such do not perform CPUID detection. So, we need to find a target other than `make ci` that works for Mozilla. My hope is that we can perhaps do something like `make -j compile-mozilla test-c-Test` or something along those lines. I need to look into it to understand what the best option is here.

Reply:
I'm happy to take whatever is best for you :slightly_smiling_face:  As long as the target successfully runs and (ideally, but not necessarily) doesn't run too many things that are not needed on top of code extraction, verification, and tests.